editor/src/clj/editor/code/view.clj

```diff
@@ -2970,7 +2970,7 @@
    {:command :toggle-breakpoint :label "Toggle Breakpoint"}
    {:command :edit-breakpoint :label "Edit Breakpoint"}])
 
-(handler/register-menu! ::menubar :editor.app-view/view-end
+(handler/register-menu! ::menubar-view :editor.app-view/view-end
```

Comment:
I had to refactor `editor.handler` ns to make built-in and extension commands more similar (e.g. to support command ids). Because of that, id handling was changed, and now you need separate registration ids (first arg to `handler/register-menu!`) for separate things. Previously, the id was scoped to location (`:editor.app-view/view-end` or `:editor.app-view/edit-end`). Now it is scoped to the handler system. Because of that, this ns has to use different registration ids for different locations.